public/css/patternfly-react.css

```diff
@@ -0,0 +1,17 @@
+/** 
+  Important: The CSS defined here should be temporary until it is 
+  entirely contributed upstream to Patternfly / Patternfly CSS.
+
+  https://github.com/patternfly/patternfly
+  https://github.com/patternfly/patternfly-css 
+*/
+ul.pf-tabrow {
+  display: flex;
+  flex-wrap: wrap-reverse;
+}
+ul.pf-tabrow > li {
+  float: none;
+}
+ul.pf-tabrow > .pf-tabrow-contents {
```

Comment:
Please remove `ul` from all the selectors. Let's just use classes for selectors when possible.